cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -1192,7 +1194,7 @@ private KafkaVersionChange getKafkaVersionChange(StatefulSet kafkaSts) {
 
                         return kafkaSetOperations.maybeRollingUpdate(sts, pod -> {
                             log.info("{}: Downgrade: Maybe patch + rolling update of {}: Pod {}", reconciliation, stsName, pod.getMetadata().getName());
-                            return true;
+                            return "Downgrade: Patch + rolling update of " + name + ": Pod " + pod.getMetadata().getName();
```

Comment:
There's no maybe at this point. The function is returning a non-null reason to roll, so the pod will be rolled. The interesting information is that the rolling is due to downgrade. Again, include that this is phase 1 of however many phases.